packages/x-codemod/src/v6.0.0/data-grid/rename-rowsPerPageOptions-prop/index.ts

```diff
@@ -0,0 +1,17 @@
+import renameProps from '../../../util/renameProps';
+import { JsCodeShiftAPI, JsCodeShiftFileInfo } from '../../../types';
+
+const componentNames = ['DataGrid', 'DataGridPro', 'DataGridPremium'];
+const props = { rowsPerPageOptions: 'pageSizeOptions' };
```

Comment:
Nitpick: You could have considered having a single codemod handling the rename of all the changed props.
Or wouldn't it have worked? 🤔

Reply:
That's actually a good point. 👍 
We discussed the approach and agreed on grouping codemods based on topic/PR because.

1. It will be easy that way to split the codemods and develop them iteratively as the PRs go by.
2. Possibly a user is heavily using a specific feature or set of features and they may want to run the codemods for those features only.
3. Same approach is being followed in the core, so they split codemods based on a specific component/set of related components.

Although to me it's fine other way around too as most of the stable mods will anyways go into preset-safe. So mostly it's not a big deal, we can group related codemods to save some clutter.